cid/utils.py

```diff
@@ -256,3 +257,11 @@ def unset_parameter(param_name):
         value = params[param_name]
         del params[param_name]
         logger.info(f'Cleared {param_name}={value}, from parameters')
+
+
+def inject_variables(source: str, variables: Dict[str, Any]) -> str:
+    """Inject variables into string"""
+    for key, value in variables.items():
+        source = source.replace(key, value)
+
+    return source
```

Comment:
We discussed not to use additional function as we have templates already, right?